services/provider/server/server.go

```diff
@@ -799,27 +810,24 @@ func (s *OCSProviderServer) ReportStatus(ctx context.Context, req *pb.ReportStat
 		return nil, status.Errorf(codes.Internal, "Failed to update lastHeartbeat payload in the storageConsumer resource: %v", err)
 	}
 
-	channelName, err := s.getOCSSubscriptionChannel(ctx)
-	if err != nil {
-		return nil, status.Errorf(codes.Internal, "Failed to construct status response: %v", err)
-	}
-
-	return &pb.ReportStatusResponse{DesiredClientOperatorChannel: channelName}, nil
+	// This will be updated as part of GetStorageConfig RPC call,
```

Comment:
"This will be updated as part of GetStorageConfig RPC call".... only in newer clients but upgraded provider will start sending empty channel name to older clients and they may never get updated.

Reply:
make sense.

Reply:
resolve.